src/index.scss

```diff
@@ -259,6 +269,11 @@ settings:
 	font-family: var(--default-font);
 }
 
+/* Change the indentation of the first line of paragraphs. Default = 0 for no indentation */
+.cm-line {
```

Comment:
Hi Troy, it seems this indent works only for Live Preview, and not for Reading view, because `.cm-line` doesn't exist in reading view. 

To make it work in Reading view, it seems you should select `.el-p` in addition to `cm-line`. So:

```
.el-p,
.cm-line {
```

Note that this behaviour would be a little inconsistent. `.el-p` targets just *paragraphs* which I'm sure is what you want, while `.cm-line` targets all lines in live preview mode regardless of type, including things like headings and images and quotes and more things you probably don't want indent.

Can I assume you've already been using this feature as you've written?